f2/src/ConfirmationPage.js

```diff
@@ -53,9 +39,6 @@ async function postData(url = '', data = {}) {
 const prepFormData = (formData, language) => {
   let contactInfo = formData.contactInfo ? formData.contactInfo : {}
   let { fullName, email, postalCode } = contactInfo
```

Comment:
You can also delete these two  lines since we don't have to mess with `contactInfo` anymore, as well as the 
```
contactInfo: {
  fullName,
  email,
  postalCode,
},
```
block in the return statement.
(it looks like this was also preventing the phone number from being submitted)

Reply:
done